src/controllers/ProductController.ts

```diff
@@ -87,7 +87,7 @@ export default class ProductController {
     try {
       const { id } = req.params;
       await ProductQuery.deleteProduct(Number(id));
-      res.json({
+      res.status(202).json({
```

Comment:
please change this to `res.sendStatus(204)` only

when you delete a product we don't return anything(no response body) and the right status for this is 204 no content